dashboards-observability/public/components/event_analytics/explorer/events_views/docViewRow.tsx

```diff
@@ -195,6 +195,8 @@ export const DocViewRow = forwardRef((props: IDocViewRowProps, ref) => {
         openTraces={openTraces}
         rawQuery={rawQuery}
         toggleSize={flyoutToggleSize}
+        backButtonExists={false}
+        onBackButtonClick={() => {}}
```

Comment:
just curious, why is `onBackButtonClick` empty? Is it a required part of DocFlyout that we have no functional use for?